src/crystal/system/win32/process.cr

```diff
@@ -190,3 +194,7 @@ private def close_handle(handle)
     raise RuntimeError.from_winerror("CloseHandle")
   end
 end
+
+private def filetime_to_f64secs(filetime)
```

Comment:
We could move this method to `Crystal::System::Time` because there's already the similar `filetime_to_seconds_and_nanoseconds`.

Reply:
`Crystal::System::Time.filetime_to_seconds_and_nanoseconds` changes the epoch from year 1601 to 1970. On the other hand this `filetime_to_f64secs` just changes integer in 100-nanoseconds to f64 in seconds. When this method is moved to `Crystal::System::Time`, I think it is better to rename the method name or add a comment in order not to misuse it. However I don't come up with its name nor comment. Do you have any idea? Otherwise, am I thinking too match?

Reply:
I fixed it as requested anyway.

Reply:
Na, that's fine. These methods are only internal API anyway. I just want them to be at the same place to see what's already there. Maybe the next API port needs something similar and would write another local helper method. Better to have related methods grouped.